controllers/etcd.go

```diff
@@ -218,8 +218,9 @@ func (r *TalosControlPlaneReconciler) auditEtcd(ctx context.Context, tcp *contro
 			// break apart the noderef name in case it's an fqdn (like in AWS)
 			machineNodeNameExploded := strings.Split(machine.Status.NodeRef.Name, ".")
```

Comment:
should we try to use the `hostname` here, not the nodename (if available?), that should fix the need to do lowercasing

Reply:
updated, use the fallback if the hostname isn't found in the address list